libindex/libindex.go

```diff
@@ -52,6 +57,26 @@ func New(ctx context.Context, opts *Opts) (*Libindex, error) {
 	// register any new scanners.
 	pscnrs, dscnrs, rscnrs, err := indexer.EcosystemsToScanners(ctx, opts.Ecosystems)
 	vscnrs := indexer.MergeVS(pscnrs, dscnrs, rscnrs)
+
+	h := md5.New()
+	var ns []string
+	m := make(map[string][]byte)
+	for _, s := range vscnrs {
+		n := s.Name()
+		m[n] = []byte(n + s.Version() + s.Kind() + "\n")
+		ns = append(ns, n)
+	}
+	if _, err := io.WriteString(h, "libindex number: 1\n"); err != nil {
```

Comment:
why? @hdonnay

Reply:
I mentioned this in person; this is a string we can change when code inside libindex itself changes.

Reply:
that was my hunch, feels like it should be somewhere/something more then an line string. My head goes to a Version constant of some sort.